index.js

```diff
@@ -30,44 +34,38 @@ function encode (buffer, number, offset) {
     buffer.writeUInt32LE(number, offset + 1)
   }
 
-  return size
+  encode.bytes = size
```

Comment:
I'm not a fan of this. But I understand the problem. You want to know what size it is but don't want to always return an object with 3 things.

imo this isn't much of an annoyance anymore because of:

```javascript
const { buffer, size } = encode(257)
```